config/locales/views/en.yml

```diff
@@ -0,0 +1,28 @@
+en:
```

Comment:
I assume strings we currently have in the controllers will also go in here? Since presumably they will end up in one view or another?

Reply:
We could do that or put them in their own controllers folder if there are a lot of them.